test.yaml.jinja

```diff
@@ -1,6 +1,7 @@
 {%- import "_macros.jinja" as macros -%}
 {%- import "_traefik1_labels.yml.jinja" as traefik1_labels -%}
 {%- import "_traefik2_labels.yml.jinja" as traefik2_labels -%}
+{%- import "_traefik3_labels.yml.jinja" as traefik3_labels -%}
```

Comment:
@josep-tecnativa  could you please confirm if, for test domains, labels are generated correctly? I see this import, but it is not used.

Reply:
He's on holidays till the second week of September.